### PR TITLE
added elysium chain as mainnet

### PR DESCRIPTION
**Pull request by farazahmadvaival**
added Elysium chain as main-net

### CODE REVIEW

File: packages/apps-config/src/api/spec/elysium.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Copyright 2017-2024 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { OverrideBundleDefinition } from '@polkadot/types/types';
+
+/* eslint-disable sort-keys */
+const definitions: OverrideBundleDefinition = {
```
Comment (TarikGul) [inline packages/apps-config/src/api/spec/elysium.ts:7]: This file needs to be imported into `index.ts`, and exported as part of the `spec`.

File: packages/apps-config/src/api/spec/elysium.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+           "v": "u64",
+           "r": "H256",
+           "s": "H256"
+        }
+      }
+    }
+  ]
+};
+
+export default definitions;
```
Comment (TarikGul) [inline packages/apps-config/src/api/spec/elysium.ts:38]: New line at end of file.

**Review by TarikGul — CHANGES_REQUESTED** [review]
Overall looks fine just needs a few added bits.

File: packages/apps-config/src/api/spec/elysium.ts (hunk 1 above)
Comment (farazahmadvaival) [reply packages/apps-config/src/api/spec/elysium.ts:7]: This file has been imported in index.ts.

**Comment by TarikGul** [comment]
Just need to run `yarn lint`

**Comment by farazahmadvaival** [comment]
i have done all steps. please verify and approve this request.

**Review by TarikGul — APPROVED** [review]

**Pull request merged**